plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/config/Config.java

```diff
@@ -0,0 +1,58 @@
+package io.jenkins.tools.pluginmanager.config;
+
+import io.jenkins.tools.pluginmanager.impl.Plugin;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Config {
+    private File pluginDir;
+    private boolean showWarnings;
+    private String jenkinsWar;
+    private List<Plugin> plugins;
+
+    public void setPluginDir(File pluginDir) {
+        this.pluginDir = pluginDir;
+    }
+
+    public File getPluginDir() {
+        return pluginDir;
+    }
+
+    public void setShowWarnings(boolean showWarnings) {
+        this.showWarnings = showWarnings;
+    }
+
+    public boolean hasShowWarnings() {
```

Comment:
Usually it is `isShowWarnings()`. Applies to methods below as well

Reply:
Will be fixed in next PR.